.github/workflows/ci.yml

```
@@ -2,9 +2,9 @@ name: Next.js CI
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "main", "dev", "new-design" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)